inference-engine/samples/speech_sample/main.cpp

```diff
@@ -708,10 +729,22 @@ int main(int argc, char *argv[]) {
 
         /** Stores all input blobs data **/
         std::vector<Blob::Ptr> ptrInputBlobs;
-        for (auto& input : cInputInfo) {
-            ptrInputBlobs.push_back(inferRequests.begin()->inferRequest.GetBlob(input.first));
+        if (!FLAGS_iname.empty()) {
+            std::string string_iname = FLAGS_oname;
```

Comment:
Is this correct? I mean `FLAGS_oname` is used to get names for both inputs and outputs.